EvidenceApi/V1/UseCase/CreateDocumentSubmissionUseCase.cs

```diff
@@ -18,19 +18,22 @@ namespace EvidenceApi.V1.UseCase
     public class CreateDocumentSubmissionUseCase : ICreateDocumentSubmissionUseCase
     {
         private readonly IEvidenceGateway _evidenceGateway;
+        private readonly IResidentsGateway _residentsGateway;
         private readonly IDocumentsApiGateway _documentsApiGateway;
         private readonly IDocumentTypeGateway _documentTypeGateway;
         private readonly INotifyGateway _notifyGateway;
         private readonly ILogger<CreateDocumentSubmissionUseCase> _logger;
 
         public CreateDocumentSubmissionUseCase(
             IEvidenceGateway evidenceGateway,
+            IResidentsGateway residentsGateway,
```

Comment:
This is not a criticism of your PR as you are just following the pattern that Hackney are using to implement clean architecture, but having this many dependencies for a class suggests a code smell, i.e. the useCase is doing too much. That's a conversation for another day though.